modules/nw-cudn-localnet.adoc

```diff
@@ -0,0 +1,79 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-localnet_{context}"]
+= Creating a ClusterUserDefinedNetwork CR for Localnet
+
+A `Localnet` topology connects the secondary network to the physical underlay. This enables both east-west cluster traffic and access to services running outside the cluster, but it requires additional configuration of the underlying Open vSwitch (OVS) system on cluster nodes.
+
+.Prerequisites
+
+* You have logged in as a user with `cluster-admin` privileges.
+
+* You have created and configured the Open vSwitch (OVS) bridge mapping to associate the physical network with the interface label created by OVS. For more information, see "Configuration for a localnet switched topology".
```

Comment:
I find this more confusing than it needs to - especially the "interface label" part. 

What do we even mean w/ that ? The purpose of the bridge mappings is to associate the logical network (OVN) with a physical Kubernetes node interface via an OVS bridge (to which said interface is connected to).

Reply:
I rewrote this to try to clarify.